lib/modules/datasource/docker/index.ts

```diff
@@ -548,6 +631,80 @@ export class DockerDatasource extends Datasource {
     return null;
   }
 
+  @cache({
+    namespace: 'datasource-docker-architecture',
+    key: (
+      registryHost: string,
+      dockerRepository: string,
+      currentDigest: string
+    ) => `${registryHost}:${dockerRepository}@${currentDigest}`,
+    ttlMinutes: 1440,
```

Comment:
I think this can be cached for at least 4 weeks.
```suggestion
    ttlMinutes: 1440 * 14,
```

Reply:
Did you mean to write 28? 14 != 4 weeks :wink:

Reply:
sure 🙈